cluster/src/main/java/io/atomix/cluster/messaging/impl/ClusterMessage.java

```diff
@@ -153,6 +153,17 @@ public static ClusterMessage fromBytes(byte[] bytes) {
         payloadBytes);
   }
 
+  static byte[] getPayload(byte[] bytes) {
```

Comment:
stripped down version of fromBytes which doesn't allocate char arrays as they were not used in the apply callbacks.